src/import_edf.jl

```diff
@@ -254,18 +249,24 @@ out as Onda annotations to `recording`.
 
 See the OndaEDF README for additional details regarding EDF formatting expectations.
 """
-function import_edf!(dataset::Dataset, edf::EDF.File, uuid::UUID=uuid4();
+function import_edf!(path, edf::EDF.File, uuid::UUID=uuid4();
                      custom_extractors=(), import_annotations::Bool=true)
     EDF.read!(edf)
-    _, recording = create_recording!(dataset, uuid)
+    file_format = "lpcm.zst"
+    signals = Any[]
     for extractor in Iterators.flatten((STANDARD_EXTRACTORS, custom_extractors))
         extracted = extractor(edf)
         extracted === nothing && continue
-        onda_signal_name, onda_signal, edf_signals = extracted
-        samples = onda_samples_from_edf_signals(onda_signal, edf_signals, edf.header.seconds_per_record)
-        store!(dataset, uuid, onda_signal_name, samples)
+        samples_info, edf_signals = extracted
+        file_path = joinpath(path, "samples", string(uuid, "_", samples_info.kind, ".", file_format))
+        samples = onda_samples_from_edf_signals(samples_info, edf_signals, edf.header.seconds_per_record)
+        signal = rowmerge(store(file_path, file_format, samples, uuid, Second(0)); file_path=string(file_path))
+        push!(signals, signal)
     end
+    signals_path = joinpath(path, "onda.signals.arrow")
```

Comment:
so `*.onda.signals.arrow` is meant to be the extension, so I'd give this a different name, or better, let the user pass in the name

or BETTER, I'd actually change `export_edf`/`import_edf` to `onda_to_edf`/`edf_to_onda` or something like that, and just return the in-memory results and leave (de)serialization up to the caller. This particular control inversion is more doable/natural in the Onda v0.12+ world

Reply:
@kleinschmidt's idea from Beacontown discussion is even better/more natural: have one function for EDF -> Onda.Samples (not a signals table - only the caller doing the serializing should be in charge of constructing a signals table), and one for EDF -> annotations table

for the latter, we should probably just force the caller to pass in the desired recording UUID as input (rather than making it an optional argument with a random default), since they'd presumably have to handle it somehow anyway

Reply:
Like we discussed in beacontown, the plan here is to 
- rename `export_edf` to `onda_to_edf` (since it already returns a `EDF.File` in memory)
- refactor `improt_edf!` into two functions, one that returns the `Samples` and one that returns the table of `Annotations`.
- leave it up to the caller to write out the `Samples` to disk somewhere of their choosing

Reply:
I've implemented this now, ready for re-review